### ISSUE TITLE
Issue on page /第五章/5.3 PyTorch修改模型.html

### ISSUE THREAD

**Issue by Sere1nz** [datawhalechina/thorough-pytorch#30]
`(layer1): Sequential(
    (0): Bottleneck(
      (conv1): Conv2d(64, 64, kernel_size=(1, 1), stride=(1, 1), bias=False)
      (bn1): BatchNorm2d(64, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
      (conv2): Conv2d(64, 64, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1), bias=False)
      (bn2): BatchNorm2d(64, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
      (conv3): Conv2d(64, 256, kernel_size=(1, 1), stride=(1, 1), bias=False)
      (bn3): BatchNorm2d(256, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
      (relu): ReLU(inplace=True)
      (downsample): Sequential(
        (0): Conv2d(64, 256, kernel_size=(1, 1), stride=(1, 1), bias=False)
        (1): BatchNorm2d(256, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
      )
    )`
请问如果我想修改layer1中的(0).conv1应该怎么获取呢？目前只知道net.layer1或者net.fc这种，再往layer1中里面的层修改应该怎么获取？谢谢

**Comment by NoFish-528** [comment]
模型的参数是以OrderedDict的形式保存，作为字典的子类，你可以使用字典的索引获得net.layer1(0)conv1 @Sere1nz
